### PR TITLE
[Prot Paladin] Update Consecration.js to Typescript and Statistic

### PR DESCRIPTION
**Pull request by hpabst**
Notes:
- Update Consecration.js to Consecration.tsx
- Change usage of StatisticBox to Statistic.
- Update protection paladin integration test to new log for changes to Consecration analyzer.

Context:
- First step of updating protection paladin analyzers to Typescript and non-deprecated components.
- This is a small task I took up to onboard myself with WA once again.

Screenshots:
- Pre-changes:
consecrationPreTS
consecrationSuggestionPreTS

- Post-changes:
consecrationPostTS
consecrationSuggestionPostTS

Testing:
- `yarn build`, `yarn lint`, `yarn test:integration`

### CODE REVIEW

File: src/parser/paladin/protection/modules/spells/Consecration.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            value={`${formatPercentage(this.pctHitsMitigated)} %`}
+            label={<Trans>Hits Mitigated w/ Consecration</Trans>}
+          />
+      </Statistic>
+    );
+  }
+
+  statisticOrder = STATISTIC_ORDER.CORE(2);
+}
+
```
Comment (Pewtro) [inline src/parser/paladin/protection/modules/spells/Consecration.tsx:68]: ``statisticOrder`` is deprecated and is already handled in ``position``

**Review by Pewtro — CHANGES_REQUESTED** [review]
One simple thing and it looks good to me after 👍🏼

File: src/parser/paladin/protection/modules/spells/Consecration.tsx (hunk 1 above)
Comment (hpabst) [reply src/parser/paladin/protection/modules/spells/Consecration.tsx:68]: Resolved in commit d2dda16f4cbee77e9957af95624b8e2a298e457d.

**Comment by emallson** [comment]
@hpabst if you resolve the merge conflicts i can merge this tonight

**Comment by hpabst** [comment]
@emallson Merge conflicts resolved and checks passed.

**Review by Sharrq — APPROVED** [review]

**Pull request merged**